### PR TITLE
Add 'placement' to TooltipRenderer, add 'cursor="default"' to Box

### PR DESCRIPTION
**Pull request by markdalgleish**
See changesets for details.

### CODE REVIEW

File: lib/components/TooltipRenderer/TooltipRenderer.treat.ts [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 });
 
 export const verticalOffsetBeforeEntrance = style({
-  transform: 'translateZ(0) translateY(-4px)',
+  transform: 'translateZ(0) translateY(4px)',
   selectors: {
-    '[data-popper-placement*=top] &': {
-      transform: 'translateZ(0) translateY(4px)',
+    '[data-popper-placement*=bottom] &': {
+      transform: 'translateZ(0) translateY(-4px)',
```
Comment (tgdev) [inline lib/components/TooltipRenderer/TooltipRenderer.treat.ts:19]: should the -4px value be a theme variable (grid?) or will it work across all themes at -4px?

Comment (markdalgleish) [reply lib/components/TooltipRenderer/TooltipRenderer.treat.ts:19]: This transition isn't following the grid, it's just a coincidence :)

**Review by tgdev — APPROVED** [review]
Nice work! X and Y positioning would be ideal (which might also open things up to responsive positioning array) but this is a great start. Thanks!

**Comment by markdalgleish** [comment]
@tgdev I did actually work on `"left"` and `"right"` placement too, but found that it's really tough to do well responsively since it's highly likely the tooltip won't fit on the screen. Decided I'd wait for an actual use case rather than guessing how best to handle it.

File: .changeset/metal-pans-breathe.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+**TooltipRenderer:** Add `placement` prop, support `top` and `bottom`. Set default placement to `top`.
+
+**EXAMPLE USAGE**
+
+```tsx
+<TooltipRenderer
+  id={id}
+  tooltip={<Text>This is a tooltip!</Text>}
+  placement="top"
```
Comment (michaeltaranto) [inline .changeset/metal-pans-breathe.md:18]: Suggest we make the example different from the default value.
```suggestion
  placement="bottom"
```

Comment (markdalgleish) [reply .changeset/metal-pans-breathe.md]: Nice catch. This wasn't the default when I wrote it.

File: lib/components/TooltipRenderer/TooltipRenderer.docs.tsx [hunk 3]
```diff
@@ -56,13 +49,78 @@ const docs: ComponentDocs = {
...
+      ),
+    },
+    {
+      label: 'Placement',
       description: (
         <Text>
-          You can use multiple text elements and layout components to create
-          more custom layouts.
+          You can set the <Strong>placement</Strong> of the tooltip to{' '}
+          <Strong>top</Strong> or <Strong>bottom</Strong>. Whichever direction
```
Comment (michaeltaranto) [inline lib/components/TooltipRenderer/TooltipRenderer.docs.tsx:65]: Worth calling out that `top` is the default here in the docs?

File: lib/components/TooltipRenderer/TooltipRenderer.docs.tsx [hunk 4]
```diff
@@ -56,13 +49,78 @@ const docs: ComponentDocs = {
...
+          you choose, the tooltip will be centred against the trigger element.
         </Text>
       ),
+      Example: ({ id }) =>
+        source(
+          <Inline space="small">
+            <TooltipRenderer
+              id={`${id}_1`}
+              placement="top"
```
Comment (michaeltaranto) [inline lib/components/TooltipRenderer/TooltipRenderer.docs.tsx:74]:
```suggestion
              placement="bottom"
```

File: lib/components/TooltipRenderer/TooltipRenderer.docs.tsx [hunk 5]
```diff
@@ -56,13 +49,78 @@ const docs: ComponentDocs = {
...
+              tooltip={<Text>The placement is “top”</Text>}
```
Comment (michaeltaranto) [inline lib/components/TooltipRenderer/TooltipRenderer.docs.tsx:75]:
```suggestion
              placement="bottom"
              tooltip={<Text>The placement is “bottom”</Text>}
```

**Review by michaeltaranto — APPROVED** [review]

**Pull request merged**